Code/Framework/AzFramework/AzFramework/Script/ScriptComponent.h

```diff
@@ -130,5 +174,12 @@ namespace AzFramework
         AZ::Data::Asset<AZ::ScriptAsset>    m_script;               ///< Reference to the script asset used for this component.
         int                                 m_table;                ///< Cached table index
         ScriptPropertyGroup                 m_properties;           ///< List with all properties that were tweaked in the editor and should override values in the m_sourceScriptName class inside m_script.
+
+        // carbonated begin (akostin/mp226-2): Add NetBindable to ScriptComponent
+#if defined(CARBONATED)
+        ScriptNetBindingTable* m_netBindingTable; ///< Table that will hold our networked script values, and manage callbacks
```

Comment:
This variable is not a static.
Could you delete it in the destructor?
And maybe protect it from the secondary initialization after delete, if it is necessary.

Reply:
Thanks!

Reply:
Another thing:
 m_netBindingTable should be set to NULL in the coonstructor